server/cluster.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"database/sql"
+
+	"github.com/mattermost/squirrel"
+	"github.com/pkg/errors"
+
+	"github.com/mattermost/mattermost/server/public/model"
+)
+
+func pingClusterDiscoveryTable(db *sql.DB, driverName, clusterName string) ([]*model.ClusterDiscovery, error) {
+	builder := squirrel.StatementBuilder.PlaceholderFormat(getQueryPlaceholder(driverName))
+
+	query := builder.Select("*").From("ClusterDiscovery").
```

Comment:
Since we are only using the hostname anyways, instead of doing `select *`, we can just get the hostname and also sort by it from the DB itself. That way, it becomes easy to compare the result slice for any changes. We don't have to do the sorting or put it in a map to compare. Just iterate the slices and break if there's a difference.

Reply:
Makes sense, this was the server code as you mentioned, I just wanted to quickly prototype. Indeed all we need is the hostname 👍